micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/ServiceRestClientConfigurationSupport.groovy

```diff
@@ -90,10 +91,17 @@ class ServiceRestClientConfigurationSupport {
         RestTemplate restTemplate = new RestTemplate(configurer.maxLogResponseChars)
         this.configureMessageConverters(restTemplate.messageConverters)
         restTemplate.requestFactory = requestFactory()
-        restTemplate.interceptors = interceptors
+        restTemplate.interceptors = filteredInterceptors()
         return restTemplate
     }
 
+    private List<ClientHttpRequestInterceptor> filteredInterceptors() {
+        if (interceptors == null) {
+            return []
+        }
+        return interceptors.findAll { !(it instanceof LoadBalancerInterceptor) }
```

Comment:
You missed KamilSz's presentation ;). `return interceptors.findAll { ... }` (without guarding `if`) is enough. See [this](https://groovyconsole.appspot.com/script/5094585248251904).